docs/census.rst

```diff
@@ -0,0 +1,78 @@
+Census
+=============
+
+********
+Overview
+********
+
+Connects to the Census API--it has been tested with the ACS and Economic Survey endpoints.
+
+.. note::
+  Authentication
+    The API requires a key that you can get `here <https://www.census.gov/data/developers.html>`_ (click on the box that says "Request a KEY")
+
+
+
+==========
+Quickstart
+==========
+
+To instantiate the Census class, either store your ``CENSUS_API_KEY`` as an environment
+variable or pass it as an argument:
+
+.. code-block:: python
+
+   from parsons import Census
+
+   # First approach: Use API key environment variables
+
+   # In Mac OS, for example, set your environment variables like so:
+   # export CENSUS_API_KEY='MY_CENSUS_KEY'
+
+   census = ActBlue()
+
+   # Second approach: Pass API key as arguments
+   actblue = ActBlue(api_key = 'MY_CENSUS_KEY')
+
+
+
+**Example 1**
+
+.. code-block:: python
+
+  from parsons import Census
+  # these variables tell the API what data we want
+  year = '2019'
+  dataset_acronym = '/acs/acs1'
+  variables = 'NAME,B01001_001E'
+  location = 'us:1'
+
+  # here's our connector
+  census = Census(api_key=acs_key)
+
+  # now pull data into a Parsons table
+  t = census.get_census(year,dataset_acronym,variables,location)
```

Comment:
Can you add a space after these commas? They didn't get linted because they're inside the docs but it doesn't hurt to keep things consistent